src/libraries/System.Net.Quic/tests/FunctionalTests/MsQuicTests.cs

```diff
@@ -352,10 +352,14 @@ public async Task UntrustedClientCertificateFails()
             }
         }
 
-        [Fact]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/99074")]
+        [ConditionalFact]
```

Comment:
```suggestion
        [ActiveIssue("https://github.com/dotnet/runtime/issues/99074", typeof(PlatformDetection), nameof(PlatformDetection.IsNotAlpine))]
```
You'd need to introduce `IsNotAlpine`, but that's trivial.